src/adcm_client/packer/naming_rules.py

```diff
@@ -88,10 +88,14 @@ def resolve_build_id(git_data, master_branches, timestamp):
                 build_id = '-' + git_data['branch'].replace("-", "_").replace("/", "_")
     else:
         build_id = '-1'
+        # timestamp
+    if timestamp_enabled:
+        build_id = build_id + '_' + timestamp
```

Comment:
To be honest this like here seams a little bit stange, what if git data and pr in git data, then version is 1.2.3-rc12345_12345 ?